preparation.sh

```diff
@@ -9,6 +9,13 @@ mv files/data.osm.pbf original.pbf
 osmconvert -v original.pbf --parameter-file=/parameter-file -o=files/data.osm.pbf
 rm original.pbf
 
+for file in $(ls changesets | grep '\.osc$'); do
+    mv files/data.osm.pbf original.pbf
+    printf "Applying changeset: $file"
+    osmconvert -v original.pbf "$file" -o=data.osm.pbf
```

Comment:
issue: this isn't putting data back in the original location
```suggestion
    osmconvert -v original.pbf "$file" -o=files/data.osm.pbf
```